src/metabase/pulse/render.clj

```diff
@@ -398,8 +398,10 @@
 
 (defn render-pulse-card
   "Render a single CARD for a `Pulse`. DATA is the `:data` from QP results (I think)."
-  [card data]
+  [card result]
   (try
+    (if (:error result) (throw (Exception. "Card has errors")))
```

Comment:
Ah ok. Been spending too much time in JavaScript land where if is the only game in town. =/